docs/sample-scripts/excel-scripts.yaml

```
@@ -1500,6 +1500,22 @@
         }
       }); 
     }
+'ExcelScript.PageLayout#setPrintComments:member(1)':
+  - |-
+    /**
+     * This script makes the comments print at the end of the worksheet
```

Consider something like...
```suggestion
     * This script prints the comments found throughout a worksheet at the end of that worksheet
```